### PR TITLE
fix: Map conversion in case of missing format

### PR DESCRIPTION
**Pull request by onuzbee**
# Fixes #

Jira: https://issues.corp.twilio.com/browse/DII-398

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: examples/java/rest/api/v2010/AwsCreator.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ public AwsCreator setTestObjectArray(final List<Object> testObjectArray){
         this.testObjectArray = testObjectArray;
         return this;
     }
-    public AwsCreator setTestAnyType(final Object testAnyType){
+    public AwsCreator setTestAnyType(final Map<String, Object> testAnyType){
```
Comment (childish-sambino) [inline examples/java/rest/api/v2010/AwsCreator.java:131]: Need a unit test that verifies a map can be passed in here.

Comment (onuzbee) [reply examples/java/rest/api/v2010/AwsCreator.java:131]: @childish-sambino  Added a test for passing the map.

File: examples/java/unit-test/rest/TwilioRestTest.java [hunk 2]
```diff
@@ -264,6 +264,30 @@ public void testObjectArrayTypeParam() {
...
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        mockRequest.addPostParam("TestAnyType", "{TestInteger=1}");
+        mockRequest.addPostParam("TestString", "AC222222222222222222222222222222");
+        when(twilioRestClient.request(mockRequest)).thenReturn(new Response("{\"account_sid\":\"AC222222222222222222222222222222\", \"sid\":\"PNXXXXY\"}", 200));
+        when(twilioRestClient.getObjectMapper()).thenReturn(objectMapper);
+        AwsCreator awsCreator = new AwsCreator("AC222222222222222222222222222222");
+        awsCreator.setTestAnyType(anyMap);
+        awsCreator.create(twilioRestClient);
```
Comment (kridai) [inline examples/java/unit-test/rest/TwilioRestTest.java:286]: fetch the response  for ` awsCreator.create(twilioRestClient)` (Account object) and assert that for not null .

Comment (onuzbee) [reply examples/java/unit-test/rest/TwilioRestTest.java]: Thanks for pointing! 
Updated!

**Review by childish-sambino — APPROVED** [review]

**Review by kridai — APPROVED** [review]

**Pull request merged**